app/src/main/res/layout/profile_chooser_fragment.xml

```diff
@@ -18,7 +18,11 @@
       android:layout_height="match_parent"
       android:background="@color/profileChooserBackground"
       android:overScrollMode="never"
-      android:scrollbars="none">
+      android:scrollbars="none"
+      app:layout_constraintBottom_toBottomOf="@+id/view"
```

Comment:
We dont have these attributes in other files and only in this file. Why is that so?

Also, if you want to add attributes then the code will change something like this:

```
    <androidx.core.widget.NestedScrollView
      android:layout_width="0dp"
      android:layout_height="0dp"
      android:background="@color/profileChooserBackground"
      android:overScrollMode="never"
      android:scrollbars="none"
      app:layout_constraintBottom_toBottomOf="parent"
      app:layout_constraintEnd_toEndOf="parent"
      app:layout_constraintStart_toStartOf="parent"
      app:layout_constraintTop_toTopOf="parent">
```

Reply:
Done